Framework/CurveFitting/test/Algorithms/FitPowderDiffPeaksTest.h

```diff
@@ -117,355 +456,90 @@ class FitPowderDiffPeaksTest : public CxxTest::TestSuite {
-
-    // 2. Get workspace
-    MatrixWorkspace_sptr dataws = boost::dynamic_pointer_cast<MatrixWorkspace>(
-        AnalysisDataService::Instance().retrieve(datawsname));
-    if (!dataws) {
-      throw runtime_error("Failed to retrieve data workspace from LoadAsii.");
-    }
-
-    return dataws;
+class FitPowderDiffPeaksTestPerformance : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static FitPowderDiffPeaksTestPerformance *createSuite() {
+    return new FitPowderDiffPeaksTestPerformance();
   }
-
-  //----------------------------------------------------------------------------------------------
-  /** Import data from a column data file by calling LoadAscii
-   */
-  void importDataFromColumnFile(string filename, string datawsname) {
-    // 1. Call LoadAscii
-    DataHandling::LoadAscii loader;
-    loader.initialize();
-
-    loader.setProperty("FileName", filename);
-    loader.setProperty("OutputWorkspace", datawsname);
-    loader.setProperty("Separator", "Space");
-    loader.setProperty("Unit", "TOF");
-
-    loader.execute();
-
-    if (!loader.isExecuted()) {
-      stringstream errss;
-      errss << "Failed to load file " << filename << " by calling LoadAsci()";
-      throw runtime_error(errss.str());
-    }
-
-    return;
+  static void destroySuite(FitPowderDiffPeaksTestPerformance *suite) {
+    delete suite;
   }
 
-  // =============================  Reflection [From File]
-  // =======================================
-  //----------------------------------------------------------------------------------------------
-  /** Create reflection table workspaces
-   */
-  DataObjects::TableWorkspace_sptr createReflectionWorkspace(int option) {
-    // 1. Create table workspace
-    DataObjects::TableWorkspace *tablews = new DataObjects::TableWorkspace();
-    DataObjects::TableWorkspace_sptr hklws =
-        DataObjects::TableWorkspace_sptr(tablews);
-
-    tablews->addColumn("int", "H");
-    tablews->addColumn("int", "K");
-    tablews->addColumn("int", "L");
-    tablews->addColumn("double", "PeakHeight");
-    tablews->addColumn("double", "TOF_h");
-    tablews->addColumn("double", "Alpha");
-    tablews->addColumn("double", "Beta");
-    tablews->addColumn("double", "Sigma2");
-    tablews->addColumn("double", "Gamma");
-
-    // 2. Add reflections and heights
-    switch (option) {
-    case 1:
-      createLaB6PG3Bank1BraggPeaksTable(hklws);
-      break;
-
-    default:
-      stringstream errss;
-      errss << "createReflectionWorkspace does not support option " << option
-            << '\n';
-      errss << "Supported options include 1 (LaB6 for PG3 bank 1). ";
-      throw invalid_argument(errss.str());
-    }
-
-    // 3. Output information
-    std::cout << "Created Table Workspace with " << hklws->rowCount()
-              << " entries of peaks.\n";
+  void setUp() {
```

Comment:
Use the `override` keyword for `setUp` and `tearDown` please. We will move the Mantid builds to Ubuntu 16.04 soon and the newer gcc creates warnings otherwise, which makes the builds fail.